README.md

```diff
@@ -63,6 +63,10 @@ https://user-images.githubusercontent.com/15919091/137049820-8ffc4bfd-43f1-41d2-
   <source src="https://user-images.githubusercontent.com/15919091/137049820-8ffc4bfd-43f1-41d2-b078-068ddaaa0c86.mp4">
 </video>
 
+### Compatibility
+
+This Contentful app works is **not compatible with Web Folder or Web Proxy Sources**. [See the imgix documentation](https://docs.imgix.com/setup/creating-sources) for a list of Sources that you can create in imgix.
```

Comment:
```suggestion
This Contentful app is **not compatible with Web Folder or Web Proxy Sources**. [See the imgix documentation](https://docs.imgix.com/setup/creating-sources) for a list of Sources that you can create in imgix.
```

Reply:
Just double checking but don't we technically support web folder sources? I recall that users just can't upload to a WF.

Reply:
Good catch, yes. I read this too quickly and didn't notice it didn't say incompatible _with upload_.

Reply:
Thanks for catching those. The changes were committed, feel free to merge.